app/models/game.rb

```diff
@@ -1,7 +1,12 @@
 class Game < ActiveRecord::Base
-  has_many :pieces, dependent: :destroy
-  validates :game_name, :presence => { :message => "Game name is required!" }
-  after_create :populate_board!
+
+	has_many :pieces, dependent: :destroy
+	has_many :users, :through => :pieces
```

Comment:
has_and_belongs_to_many establishes a direct association that is used when there isn't a join table, we have a join table in the pieces table, so that isn't the optimal solution for us. But I am going to remove the :through associations in both User.rb and Game.rb because I agree we don't need that added complexity.